internal/util/toml.go

```diff
@@ -56,8 +56,8 @@ func decodeErrFromTOMLErr(e *toml.DecodeError, path AbsolutePath) *DecodeError {
 	}
 }
 
-const invalidTOMLCode types.ErrorCode = "invalidTOML"
-const unknownTOMLKeyCode types.ErrorCode = "unknownTOMLKey"
+const InvalidTOMLCode types.ErrorCode = "invalidTOML"
+const UnknownTOMLKeyCode types.ErrorCode = "unknownTOMLKey"
```

Comment:
These are the errors which definition should be shared between client and agent.